internal/runners/packages/import.go

```diff
@@ -138,7 +138,7 @@ func (i *Import) Run(params *ImportRunParams) error {
 	pg = nil
 
 	// Solve the runtime.
-	rtCommit, err := bp.FetchCommit(latestCommit, proj.Owner(), proj.Name(), nil)
```

Comment:
nit: Perhaps `latestCommit` should be renamed to `localCommit` as after we stage a commit this name is no longer valid.